src/tools/tidy/src/deps.rs

```diff
@@ -252,6 +255,12 @@ const PERMITTED_RUSTC_DEPENDENCIES: &[&str] = &[
     "winapi-i686-pc-windows-gnu",
     "winapi-util",
     "winapi-x86_64-pc-windows-gnu",
+    "windows-sys",
+    "windows_aarch64_msvc",
+    "windows_i686_gnu",
+    "windows_i686_msvc",
+    "windows_x86_64_gnu",
+    "windows_x86_64_msvc",
```

Comment:
Okay I don't understand where these deps came from. I'm pretty sure there's gotta be some pebkac going on rn but I don't get how these produced an error how, they appear to have already been part of the Cargo.toml

Gonna try removing...

Reply:
parking_lot 0.12 now depends on windows-sys. Note that this list here is only for dependencies of the standard library and rustc, not for other tools despite the fact that those use the same lockfile.